### PR TITLE
Convert body from string to bytes

### PR DESCRIPTION
**Pull request by kodevyas**

### CODE REVIEW

File: etsy/_core.py [hunk 1]
```diff
@@ -288,7 +290,7 @@ def _get(self, http_method, url, **kwargs):
 
         if http_method == 'GET':
             url = '%s%s?%s' % (self.api_url, url, urlencode(kwargs))
-            body = ''
+            body = ''.encode('utf-8')
```
Comment (Anmol-Gulati) [inline etsy/_core.py]: what about the following else conditions?
Make it outside of if condition
and use isinstance to check it's already bytes or not

**Review by Anmol-Gulati — CHANGES_REQUESTED** [review]

File: etsy/_core.py (hunk 1 above)
Comment (kodevyas) [reply etsy/_core.py]: @Anmol-Gulati We are directly assigning empty string then whats the need to check whether it is byte of not

Comment (prakashpp) [reply etsy/_core.py]: What was the error before this patch? Can you post stacktrace or something?

**Review by prakashpp — COMMENTED** [review]
What was the error before this patch? Can you post stacktrace or something?

File: etsy/_v2.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
-import urllib.request, urllib.parse, urllib.error  # noqa
+import urllib.request
+import urllib.parse
+import urllib.error  # noqa
```
Comment (sayatul) [inline etsy/_v2.py:3]: why noqa? can't we remove if not in use

**Review by sayatul — CHANGES_REQUESTED** [review]

File: etsy/_core.py [hunk 3]
```diff
@@ -307,6 +309,9 @@ def _get(self, http_method, url, **kwargs):
             body = urlencode(kwargs)
             content_type = 'application/x-www-form-urlencoded'
 
+        if not isinstance(body, bytes):
```
Comment (tarunbhardwaj) [inline etsy/_core.py]: check for string instance.

**Review by tarunbhardwaj — CHANGES_REQUESTED** [review]

**Review by tarunbhardwaj — APPROVED** [review]

**Pull request merged**